pkg/render/logstorage/externalelasticsearch/externalelasticsearch.go

```diff
@@ -112,3 +121,80 @@ func (e externalElasticsearch) oidcUserRoleBinding() client.Object {
 		},
 	}
 }
+
+func (e externalElasticsearch) elasticsearchRolesAndBindings() ([]*rbacv1.Role, []*rbacv1.RoleBinding) {
```

Comment:
This is very similar to what is in pkg/render/logstorage.go, should the one there be changed to an exported "standalone" function that can be called in the externalelasticsearch code too? Maybe with a comment that it is utilized in externalelasticsearch and any changes to it need to be considered for the external es point of view.

Reply:
I will look into maybe adding somewhere common. I could not find a common file. Maybe I need to create one.

Reply:
Seems like it is not possible at them moment. Cycle import. I will add some comments to explain this.

Reply:
Added the comments.